pages/web_cloud/web_hosting/sql_eos_eol/guide.es-es.md

```diff
@@ -10,11 +10,34 @@ updated: 2024-07-11
 
 Los productos cubiertos por estos anuncios de fin de venta y fin de vida son los servicios de base de datos de SQL Web Hosting, a los que se puede acceder a través de la red Web Hosting. Para más información, consulte la [política de fin de vida útil de las bases de datos administradas](/pages/web_cloud/web_cloud_databases/eol-policy).
 
-|Versión|Anuncio de fin de vida|Fin de venta|Fin de soporte|
+|Versión del DBMS|Anuncio de fin de vida|Fin de venta|Fin de soporte|
 |---|---|---|---|
 |MySQL 5.7|2023-11-16|2024-02-16|2024-05-17|
 |MySQL 8.0|Por determinar|Por determinar|Por determinar|
 
+> [!primary]
+>
+> Los servicios de bases de datos SQL incluidos con los planes de hosting de OVHcloud no pueden actualizarse directamente desde el área de cliente de OVHcloud ni a través de la base de datos al final de la venta/vida.
+>
+
+Si desea anticiparse al final de la vida o realizar las acciones manualmente, deberá realizar las siguientes acciones:
+
+- Caso n° 1: dispone de una sola base de datos incluida con su plan de hosting:
+- Compruebe que el contenido de la base de datos es compatible con un DBMS más reciente.
```

Comment:
Ths list is not the same here as the other versions

Reply:
done